src/structures/MessageMentions.js

```diff
@@ -86,6 +87,29 @@ class MessageMentions {
      * @private
      */
     this._channels = null;
+
+    /**
+     * Crossposted channel data.
+     * @typedef {Object} CrosspostedChannel
+     * @property {string} channelID ID of the mentioned channel
+     * @property {string} guildID ID of the guild that has the channel
+     * @property {string} type Type of the channel
+     * @property {string} name The name of the channel
+     */
+
+    /**
+     * An array of crossposted channels
+     * @type {?CrosspostedChannel[]}
```

Comment:
Shouldn't this be a `?Collection<Snowflake, CrosspostedChannel>`?
Although if available I think we should include the actual channel, so `?Collection<Snowflake, CrosspostedChannel | NewsChannel>`.

Reply:
A collection would be useful, but having the type be `CrosspostedChannel | NewsChannel` would be a bit confusing (as CrosspostedChannel isn't a true "channel" like the NewsChannel)